src/main/java/seedu/address/logic/Messages.java

```diff
@@ -48,9 +52,17 @@ public static String format(Person person) {
                 .append(person.getAddress())
                 .append("; Tags: ");
         person.getTags().forEach(builder::append);
+        try {
+            String roleSpecificInfo = getRoleSpecificInfoString(person);
+            builder.append("; ").append(roleSpecificInfo);
+        } catch (CommandException e) {
+            builder.append("; Role info unavailable: ").append(e.getMessage());
+        }
```

Comment:
Using try and catch here seems redundant since no `CommandException` will be thrown?

Reply:
Sry overlooked this part. New pr pushed